webserver/webserver/src/main/java/io/helidon/webserver/ClassPathContentHandler.java

```diff
@@ -134,14 +133,7 @@ private ExtractedJarEntry extractJarEntry(URL url) {
         }
     }
 
-    private Instant getLastModified(URL jarUrl) throws IOException {
-        // Decode potentially encoded parameters
-        String path = URI.create(jarUrl.getPath()).getPath();
-        int jarDelimIdx = path.indexOf("!/");
-        if (jarDelimIdx > -1) {
-            path = path.substring(0, jarDelimIdx);
-        }
-
+    private Instant getLastModified(String path) throws IOException {
```

Comment:
Maybe this be using URI first and then converting it to a Path using Paths.get(URI).

Reply:
I am not sure what exactly do you mean here.

Reply:
This url variable which is present there contains path to the file in jar archive.
Example:
jar:file:/D:/...../helidon/examples/microprofile/mp1_1-static-content/target/helidon-examples-microprofile-mp1_1-static-content-0.11.0-SNAPSHOT.jar!/WEB/resource.html

We need only the path to jar file itself. I don't like this type changing also, but I do believe that this is the simplest solution.

Reply:
This is not going anywhere.

Reply:
I feel that depending on a java method that claims to return path to the file is better than parsing a URI and locating the jar path from it.
Also creating a new issue #298 inspired by analysis of this class